pkg/metrics/metrics.go

```diff
@@ -0,0 +1,36 @@
+package metrics
```

Comment:
I think we should reuse instead the existing https://github.com/apache/camel-k/blob/main/pkg/controller/integration/metrics.go